mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/v1/BaseRoute.java

```diff
@@ -332,6 +332,7 @@ protected  <T> Route withFuture(CompletableFuture<T> tFuture) {
 
                     if (throwable instanceof RequestThrottledException) {
                         MasterApiMetrics.getInstance().incrementResp4xx();
+                        MasterApiMetrics.getInstance().incrementThrottledRequestCount();
```

Comment:
nit: can we log something as well?

Reply:
like? (I am going to add this to master dash)

Reply:
sure, sometimes it's just easy to spot issues when opening radar and filter by errors. up to you ;) .